src/pages/verify/integrations/javascript.mdx

```diff
@@ -0,0 +1,292 @@
+<div id="result"></div>
+...
+
+<script type="text/javascript">
+  document
+    .getElementById('login-button-popup')
+    .addEventListener('click', () => {
+      criiptoAuth.popup
+        .authorize({
+          width: 300,
+          height: 400,
+          redirectUri: 'http://localhost:8000/example/index.html',
+          acrValues: 'urn:grn:authn:dk:nemid:poces',
+        })
+        .then((result) => {
+          console.log(result, result.id_token, result.claims);
+          document.getElementById('result').textContent =
+            JSON.stringify(result);
+        });
+    });
+</script>
+```
+
+- `width (optional, number)`: Popup width
+- `height (optional, number)`: Popup height
+- For `redirectUri` and `acrValues`, see [authorization parameters](#authorization-parameters)
+
+### Authorization with redirect
+
+Will redirect the user to the Criipto Verify login page.
+
+#### authorize
+
+Redirects the users browser tab to the authorization url. After authorization, the user will be redirected back to the provided `redirectUri`.
+The `redirectUri` should be [whitelisted in the "Callback URLs" section](#register-callback-URLs) in your Criipto application settings.
+
+```html
+<!-- index.html -->
+...
+<button id="login-button-redirect">Login</button>
+<div id="result"></div>
+...
+
+<script type="text/javascript">
+  document
+    .getElementById('login-button-redirect')
+    .addEventListener('click', () => {
+      criiptoAuth.redirect.authorize({
+        redirectUri: 'http://localhost:8000/example/index.html',
+        prompt: 'login',
+      });
+    });
+</script>
+```
+
+`prompt: 'login'` will force the user to re-authenticate even if they have already logged in. See [authorization parameters](#authorization-parameters) for more information.
+
+#### match
+
+Will check for the `code` or `id_token` parameter in the URL query parameters or hash of the current `window.location`.
+
+- If the `id_token` parameter is present, it will return the result.
+- If the `code` parameter is present, it will perform a PKCE token exchange and return the result. (TODO: PKCE seciton).
+
+As soon as the user has been redirected back to your application after authentication, you can call the `match` method to get the `id_token` or `code` parameter.
+
+```js
+const match = await criiptoAuth.redirect.match();
+console.log(match.code);
+console.log(match.id_token, match.claims);
+```
+
+Using `claims` to display the user name:
+
+```html
+<!-- index.html -->
+...
+<h2>Hello, <span id="user-name">user</span></h2>
+...
+
+<script type="text/javascript">
+  criiptoAuth.redirect
+    .match()
+    .then((result) => {
+      if (result && result.claims) {
+        const userName = document.getElementById('user-name');
+        userName.textContent = result.claims.name;
+      }
+    })
+    .catch((error) => {
+      console.log(error);
+    });
+</script>
+```
+
+### Responsive authorization: popup or redirect based on the screen size
```

Comment:
Remove this section, I don't like this API very much so no reason to promote it.